tests/fuzz/s2n_openssl_diff_pem_parsing_test.c

```diff
@@ -120,6 +124,7 @@ int LLVMFuzzerTestOneInput(const uint8_t *buf, size_t len)
         /* return -1; */
     }
 
+    s2n_stuffer_free(&in);
```

Comment:
I think you still want the GUARD() here.

Reply:
Updated.